README.md

```diff
@@ -103,20 +106,26 @@ render(welcome())
 [buble]: https://github.com/Rich-Harris/buble
 [buble-jsx]: https://buble.surge.sh/guide/#jsx
 
-## Overview
+## Specifications
```

Comment:
I'm not really happy with "Specifications", but it's slightly too detailed for "Overview"